### PR TITLE
Aerogear 3054 keycloak

### PR DESCRIPTION
**Pull request by maleck13**

### CODE REVIEW

**Comment by maleck13** [comment]
ping @david-martin this is the way the pages for the services will look before the docs are done

File: services/keycloak/index.html [hunk 1]
```diff
@@ -1,5 +1,31 @@
 ---
 layout: index
-title: Aerogear Keycloak
+title: Security
```
Comment (david-martin) [inline services/keycloak/index.html]: Identity Management

File: services/crc/index.html [hunk 2]
```diff
@@ -3,3 +3,22 @@
 title: Aerogear CRC
 section: mobile-services
 ---
+
+
+<h2 class="section-header"><i class="fa fa-plug" aria-hidden="true"></i>
+    Custom Runtime Connector</h2>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit ante. Proin cursus sagittis vehicula. Nulla mattis id erat non sagittis. Nullam ut ligula venenatis, ultrices nulla a, convallis sem. Interdum et malesuada fames ac ante ipsum primis in faucibus. Vestibulum tincidunt, erat quis tincidunt bibendum, est sapien viverra erat, et dictum felis dolor vel nulla. Sed hendrerit tincidunt ligula id mollis. Nam iaculis lectus risus, sed suscipit dui tempus vitae. Pellentesque nec ultrices risus, sit amet vehicula nisi. </p>
```
Comment (david-martin) [inline services/crc/index.html]: fyi @pb82 Here is the site placeholder for an overview of the custom runtime service.
The content will probably be similar/reused for the docs site overview of this service.

Comment (pb82) [reply services/crc/index.html]: @david-martin thanks, bookmarked! @wtrocki has already provided some details about the CRCs in https://issues.jboss.org/browse/AEROGEAR-2988

File: services/crc/index.html [hunk 3]
```diff
@@ -3,3 +3,22 @@
...
+
+
+<h3>Getting Started</h3>
+<p>This guide will show you how to set up a Custom Runtime Connector and consume it from your mobile client.
+    <a href="https://github.com/aerogearcatalog/custom-runtime-connector-apb/blob/master/docs/provision.adoc">Geting started with the Custom Runtime Connector</a>
```
Comment (david-martin) [inline services/crc/index.html]: @finp @pb82 Links to docs will need to be updated once the docs site is published

Comment (pb82) [reply services/crc/index.html]: Captured in JIRA: https://issues.jboss.org/browse/AEROGEAR-3072

Comment (odra) [inline services/crc/index.html]: Do you think it is a good idea to set `target="_blank"` for external links so website visitors won't  leave the website?

**Comment by maleck13** [comment]
closed in favour of https://github.com/aerogear/aerogear.org/pull/730

**Pull request closed without merging**